packages/data-transport-layer/src/services/l1-ingestion/handlers/sequencer-batch-appended.ts

```diff
@@ -116,12 +116,12 @@ export const handleEventsSequencerBatchAppended: EventHandlerSet<
           batchIndex: extraData.batchIndex.toNumber(),
           blockNumber: BigNumber.from(context.blockNumber).toNumber(),
           timestamp: BigNumber.from(context.timestamp).toNumber(),
-          gasLimit: BigNumber.from(extraData.gasLimit).toString(),
```

Comment:
What do you think about deleting top level `gasLimit`, `target` and `value`. The raw can be at `data` (tbh i'd prefer calling it something that isn't confused with calldata) which deduplicates all of the fields in `decoded`

Reply:
This could be done in a follow up PR